### DISCUSSION TITLE
How do I use this?

### DISCUSSION THREAD

**Discussion by Nicklenselink** [v1993/linuxmotehook2#4] [Q&A] [answered]
I've just installed all dependencies and ran the commands under quick build guide. How do I now actually start the server and use it in Cemu or Yuzu? I'm pretty new to Linux so not sure what to look for

**Comment by martinprad0** [comment, marked as answer]
Since I had this question a few hours ago, I believe that my answer might not be useful for you right now, but might help more users:
In the first place, you have to connect your WiiMote through Bluetooth. Then, in the folder you installed `linuxmotehook2` you may find a file called `ExampleConfig.ini`. You may create your own `.ini` config file following the instructions [here](https://github.com/v1993/linuxmotehook2/wiki). After doing that, start the linuxmotehook2 client by using the following command on the terminal.
```
cd path/to/linuxmotehook2
linuxmotehook2 ExampleConfig.ini
```

If everything goes as intended, then you will get the following output:

```
Found wiimote 0x0023CCAC45BF - connecting... done!
```
Otherwise, you will get a message like `Wiimote 0x0023CCAC45BF not in config - skipping` which means the config file needs to be edited again.

Now, open Cemu GUI and go to `Options > Input Settings`. Then, pick any controller you want to emulate and do the following:
`Controller > + >`
1)  `API > DSU Controller`
2) `Controller > Controller 1`
Then, do steps 1) and 2) again to add Controller 2 [DSU Controller]

- Controller 1 is the Wiimote
- Controller 2 is the Nunchuck.

You may map your keys by choosing between the 2 controllers, but don't worry because both will work when you open your games if you mapped some keys with the Wiimote and some keys with the Nunchuck.

**Reply by v1993** [reply]
It's also not quite a correct answer, although thanks for your effort regardless - as [our wiki](https://github.com/v1993/linuxmotehook2/wiki) points out these days

> All values have reasonable defaults, so it's advised to avoid setting them unless you know what you're doing.

P.S. Use ``` to escape multiline code blocks. And "Wiimote number" is its MAC address.

**Reply by martinprad0** [reply]
I edited the answer with the recommendations you gave me. I hope it's more clear now.

**Reply by v1993** [reply]
I'd personally suggest starting with an *empty* file and then adding sections as you need - maybe I should add that to example config's header ([done](https://github.com/v1993/linuxmotehook2/commit/cf97f46dea443bcd3903d5404e678b79c1ec5659))... Also, why link to `port` subsection of config wiki page?

**Reply by martinprad0** [reply]
Man I feel like a total newbie, but I have to thank you for your patience. I edited my answer again.

> Also, why link to port subsection of config wiki page?

I didn't notice sorry.

**Reply by v1993** [reply]
No worries - this should be pretty good now.

**Reply by kirbeh73** [reply]
I run the "cd path/to/linuxmotehook2
linuxmotehook2 ExampleConfig.ini" in terminal and it says "bash: cd: path/to/linuxmotehook2: No such file or directory" while I am literally looking at it. On SteamOS btw
